src/xenia/ui/window.cc

```diff
@@ -17,6 +17,8 @@
 #include "xenia/base/logging.h"
 #include "xenia/ui/imgui_drawer.h"
 
+#include "xenia/base/profiling.h"
+
```

Comment:
Maybe an option for the user to toggle showing the FPS number in the titlebar.
Since applications like OBS won't be able to hook due to the title kept changing. 1/2
Note: i can't start a review at line 15 so when merging include cvar before logging.
```suggestion
#include "xenia/base/cvar.h"

DEFINE_bool(fps_titlebar, true, "Show FPS in titlebar", "General");